files/scripts/healdata/heal-cedar-data-ingest.py

```diff
@@ -16,13 +64,13 @@
 
 if not args.directory:
     print("Directory ID is required!")
-    exit(1)
+    sys.exit(0)
```

Comment:
These should probably exit with the code 1 as this is an error

Reply:
yeah ideally we should, but since this is a job if we exit with 1 then k8s will retry it, creating a bunch of doomed job pods...

Reply:
I guess maybe I can put it to exit with 1 but update the job retry policy?

Reply:
That does make sense, that we don't want the job to retry. But I think we would rather see `error` instead of `completed` on the job run. It would make it easier to know something went wrong